pkgs/development/libraries/libroadrunner/default.nix

```diff
@@ -0,0 +1,68 @@
+{ lib, stdenv, callPackage, fetchFromGitHub, cmake, gcc, boost, eigen, libxml2, mpi, python3, petsc, llvm_13, swig4 }:
+
+let
+  pythonDeps = with python3.pkgs; [
+    numpy
+    matplotlib
+  ];
+
+  roadrunner-deps = callPackage ./roadrunner-deps.nix { };
+
+  version = "2.2.0";
+
+  roadrunner = stdenv.mkDerivation rec {
+    inherit version;
+    pname = "roadrunner";
+
+    src = fetchFromGitHub {
+      owner = "sys-bio";
+      repo = pname;
+      rev = "v${version}";
+      sha256 = "sha256:0y7r0f1fm6xw57yf9wximw7vll1bpr7fsx477kraabskvqyfpg7r";
+    };
+
+    patches = [
+      ./0001-Change-numpy-version.patch
+    ];
+
+    buildPhase = ''
+      export LD_LIBRARY_PATH=$PWD/lib
+      make -j$(nproc)
+    '';
+
+    cmakeFlags = [
+      "-DCMAKE_BUILD_TYPE=Release"
+      "-DBUILD_RR_PLUGINS=ON"
+      "-DRR_DEPENDENCIES_INSTALL_PREFIX=${roadrunner-deps}"
+      "-DLLVM_INSTALL_PREFIX=${llvm_13}"
+      "-DBUILD_TESTS=ON"
+      "-DBUILD_PYTHON=ON"
+      "-DSWIG_EXECUTABLE=${swig4}/bin/swig"
+      "-DPython_ROOT_DIR=${python3}"
```

Comment:
That will probably not work if it tries to write relative to there.

Does that bleed into the python package?

Reply:
It doesnt try to write there